src/main/java/com/box/sdk/BoxMultipartRequest.java

```diff
@@ -120,7 +120,7 @@ protected void writeBody(HttpURLConnection connection, ProgressListener listener
                 this.writeOutput(entry.getValue());
             }
 
-            this.writePartHeader(new String[][] {{"name", "filename"}, {"filename", this.filename}},
+            this.writePartHeader(new String[][] {{"name", "file"}, {"filename", this.filename}},
```

Comment:
Is there a reason for "name=file" in the Content-Disposition header?  AFAIK, it's not part of the standard.  Maybe legacy support?  If no good reason, then perhaps we should remove it.

Also, it appears the writePartHeader still has escaping problems:
https://github.com/box/box-java-sdk/blob/master/src/main/java/com/box/sdk/BoxMultipartRequest.java#L177-L182

Line 181 in particular is directly writing user-submitted data to a HTTP header.  That could be abused.  We should URL-encode the value.

See:  http://stackoverflow.com/questions/93551/how-to-encode-the-filename-parameter-of-content-disposition-header-in-http/6745788#6745788